### PR TITLE
Layton's FizzBuzz Script in Python

### PR DESCRIPTION
**Pull request by laytonmccafferty**
- creates FizzBuzz script for Python
 - creates shell program to run FizzBuzz script via command line arguments
 - adds README.md for accurate utilization of program
 - adds index.html so that this FizzBuzz has a webpage
 - adds the link to my FizzBuzz webpage under the general FizzBuzz wepbage

### CODE REVIEW

**Comment by laytonmccafferty** [comment]
> I ran your code and it looked good to me.

did you pull it down to your machine or just review or both? (+1 for reviewing and +10 if you ran the code on your computer and it worked + including the evidence)

File: fizzbuzz/python/laytonmccafferty/README.md
Comment (Tech-Wizz) [inline fizzbuzz/python/laytonmccafferty/README.md:1]: Really Good!!

File: fizzbuzz/python/laytonmccafferty/fizzbuzz.py
Comment (Tech-Wizz) [inline fizzbuzz/python/laytonmccafferty/fizzbuzz.py:1]: Works correctly

**Review by Tech-Wizz — APPROVED** [review]
I pulled it down and tested it and it all works correctly.

**Review by Siertischek — COMMENTED** [review]
+1 Code looks good to me

**Pull request merged**